### PR TITLE
fix: api client modal search

### PR DESCRIPTION
**Pull request by antlio**
this pr fixes the api reference search being display on hotkey pressed while api client modal is open in order to displays its own.

### CODE REVIEW

**Review by marclave — APPROVED** [review]
very smart fix @antlio ! just needs a changeset then we can merge :)

File: packages/api-reference/src/components/api-client-bus.ts [hunk 1]
```diff
@@ -5,3 +5,4 @@ const apiClientBusKey: EventBusKey<OpenClientPayload> = Symbol()
 
 /** Event bus to open the API Client */
 export const apiClientBus = useEventBus(apiClientBusKey)
+export const modalStateBus = useEventBus('modalState')
```
Comment (marclave) [inline packages/api-reference/src/components/api-client-bus.ts:8]: lets call this searchModalState

**Review by marclave — CHANGES_REQUESTED** [review]

File: packages/api-reference/src/components/api-client-bus.ts (hunk 1 above)
Comment (antlio) [reply packages/api-reference/src/components/api-client-bus.ts:8]: being the api client modal state, should we still use search here?

Comment (marclave) [reply packages/api-reference/src/components/api-client-bus.ts:8]: ah lets just keep it nvm : )

**Review by marclave — APPROVED** [review]

**Pull request merged**